README.md

```diff
@@ -58,7 +58,7 @@ The Ginkgo HIP module has the following __additional__ requirements:
 * _ROCm 4.5+_
 *    the HIP, hipBLAS, hipSPARSE, hip/rocRAND and rocThrust packages compiled with either:
     * _AMD_ backend (using the `clang` compiler)
-    * _9.2 <= CUDA < 11_ backend
```

Comment:
As a sidenote, we would need to check but on the hipSPARSE side they keep adding ifdefs for larger CUDA version. Maybe they caught up to 12 in the last versions.

https://github.com/ROCmSoftwarePlatform/hipSPARSE/blob/develop/library/src/nvidia_detail/hipsparse.cpp#L974